### PR TITLE
openstack: set 'Non-SSL' security protocol by default

### PR DESCRIPTION
**Pull request by vrutkovs**
{{pytest: cfme/tests/cloud/test_providers.py --use-provider 'rhos7-ga' -v}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit ac353c858df41dbcb4fb9e2b9200d5606c37a808:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit ff940d2ed15db95254db8447ea755aa300cb49fd:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/cloud/provider.py [hunk 1]
```diff
@@ -53,6 +53,7 @@
             }
         )),
         ("api_version", AngularSelect("api_version"), {"appeared_in": "5.5"}),
+        ('sec_protocol', {version.LOWEST: None, '5.5': AngularSelect("security_protocol")}),
```
Comment (psav) [inline cfme/cloud/provider.py]: Probably better to do like the line above here, I can't recall what setting it to ```None``` will do. It _should_ be OK, but may have undesired consequences.

**Comment by dajoRH** [comment]
Lint report for commit 5ac2a1916bb6818daf699fb963250d30604d64f5:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 31bbf2fc1153c23bd8f0255f70cc37d89b88350b:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 30d0a19519dd09e6012069bc9bff15a3d8066d0c:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**